crates/indexer/src/geyser/client.rs

```diff
@@ -92,4 +95,34 @@ impl Client {
             })
             .await
     }
+
+    /// Dispatch a POST reqwest to Dialect
+    pub async fn dispatch_dialect(&self) -> reqwest::Result {
+        enum MessageType {
+            NftOffer,
+        }
+
+        struct MessageData {
+            address: String,
+        }
+
+        struct Message {
+            msgType: MessageType,
+            data: MessageData,
+        }
+
+        let resp = self
+            .http
+            .dialect_push
+            .new()
+            .post("")
+            .body(Message {
+                msgType: MessageType::NftOffer,
+                data: MessageData {
+                    address: "1111111111111111111111".to_string(),
+                },
+            })
+            .send()
+            .await?;
```

Comment:
http is misleading for this client. its actually a rabbitmq client.

you aren't looking to add another producer. its something more like the db but preconfigured http_sender.

Reply:
hm, okay :+1: